Extend/Controller/Admin/PaymentMain.php

```diff
@@ -63,6 +72,7 @@ public function render()
             // and an error message shown in the frontend
             Helper::addToViewData($this, [
                 'bConfigNotValid' => ($sFnc === 'save' || $sFnc === 'addfield') && !$this->_bIsSavePossible,
+                'bCurrencySettings' => $this->_bCurSettingInfo,
```

Comment:
Same as above: I would also prefer a more descriptive name as "CurrencySettings" does not provide a lot of insight. It could be the same (improved) variable name if appropriate.

Reply:
👍